src/portable/destroy.rs

```diff
@@ -100,6 +101,9 @@ fn do_destroy(options: &Destroy) -> anyhow::Result<()> {
     }
     if paths.credentials.exists(){
         found = true;
+        if let Err(e) = crate::cloud::ops::try_to_destroy(&paths.credentials, opts) {
+            print::warn(format!("Failed to destroy EdgeDB Cloud instance: {:#}", e));
```

Comment:
I'm not sure this should be a warning. Maybe when `--force` is set you can ignore failure and remove the file. But most of the time we want to keep the credentials on network error or similar.